src/_vars.scss

```diff
@@ -5,9 +5,11 @@
 // Ensure that the relative path (i.e. `../../`, etc.) is correct when importing from a nested
 // component.
 
-// For use in a .js file, see `src/theme.js`.
+// For use in a .ts file, see `src/theme.ts`.
 $primary-blue: #014983;
+$primary-blue-tint: #016bc0;
 $primary-cyan: #00aeef;
+$primary-cyan-shade: #0098d0;
```

Comment:
It looks like you're only using these colors for the button hover shades, but MUI will take care of that for you if you use the built-in color props as mentioned in my other comment.

Reply:
So please remove these essentially redundant colors.

Reply:
As mentioned above, these are needed to tint contained Icon Buttons on hover.

However, there is still some inconsistency to clean up here. Why is the blue a `tint` but the cyan is  `shade`?

Did you have a specific reason for darkening the blue on hover but brightening the cyan?

If we are going to add these as global colors, they at least need to be consistent and clearly named.

Reply:
Answering the last question first:
It was more of a design choice where the `cyan` was too bright to make brighter on hover (just hurt my eyes) and `blue` was too dark to darken on hover (just not enough contrast, and the problem is exacerbated by its proximity to other `blue` items), thus I thought a tint would give it the necessary _pop_ for visual clarity. 

Regarding why it's a tint and cyan is a shade: the blue tint is literally a lighter version (higher up on theme.tx) and the cyan shade is a darker version. Thus, it just made more sense to me to name the blue a tint since it's not a shade of `primary-blue`

Reply:
> Answering the last question first: It was more of a design choice where the `cyan` was too bright to make brighter on hover (just hurt my eyes) and `blue` was too dark to darken on hover (just not enough contrast, and the problem is exacerbated by its proximity to other `blue` items), thus I thought a tint would give it the necessary _pop_ for visual clarity.

That's what I figured, thanks.

> Regarding why it's a tint and cyan is a shade: the blue tint is literally a lighter version (higher up on theme.tx) and the cyan shade is a darker version. Thus, it just made more sense to me to name the blue a tint since it's not a shade of `primary-blue`

Firstly, it's not obvious that "tint" and "shade" mean something different here - they seem like arbitrary synonyms rather than a meaningful technical distinction.

Secondly, both of these colors are shades of the respective base color in the sense that they appear in the `blueShades` and `cyanShades` list in our theme. The `$primary-blue-tint` color is [`gordonColors.primary.blueShades.300`](https://github.com/gordon-cs/gordon-360-ui/blob/068ca7ae435a0be1c24d7f37ac8b8a2848cc6f38/src/theme.ts#L12), and the `$primary-cyan-shade` is [`gordonColors.secondary.cyanShades.600`](https://github.com/gordon-cs/gordon-360-ui/blob/068ca7ae435a0be1c24d7f37ac8b8a2848cc6f38/src/theme.ts#L33). In my opinion, these should be called `$primary-blue-300` and `$primary-cyan-600` for full clarity.

Reply:
In respect to the current `theme.ts` I agree that naming it `$primary-blue-___` & `primary-cyan-____` would make the most sense. I'll make that change. 👍🏼 

However, just as a personal note, looking back at `theme.ts`, the term `shade` is not accurate for half those values. Shades are meant to be explicitly darker than a specified color as in art they're used as mixtures of black + color. Whereas, tints are meant to be the opposite, being white + color. So for pure terminology accuracy [gordonColors.primary.blue](https://github.com/gordon-cs/gordon-360-ui/blob/068ca7ae435a0be1c24d7f37ac8b8a2848cc6f38/src/theme.ts#L7) should be [gordonColors.primary.blueShades.50](https://github.com/gordon-cs/gordon-360-ui/blob/068ca7ae435a0be1c24d7f37ac8b8a2848cc6f38/src/theme.ts#L9). Though, obviously it's the center hue since it would save time and characters to not have to specify `primary.blueShades.500` so I get that.

Reply:
> However, just as a personal note, looking back at `theme.ts`, the term `shade` is not accurate for half those values. Shades are meant to be explicitly darker than a specified color as in art they're used as mixtures of black + color. Whereas, tints are meant to be the opposite, being white + color.

This is true when using "shade" and "tint" as technical terms in the fields of art and color theory. However, it is not true in general English usage. More importantly for this context, it is common practice in UI Design to refer to the "shades" (sometimes also variants) of a theme color, where "shade" has the meaning "a color, especially with regard to how light or dark it is or as distinguished from one nearly like it." (cf. https://www.google.com/search?q=shade+meaning).

> So for pure terminology accuracy [gordonColors.primary.blue](https://github.com/gordon-cs/gordon-360-ui/blob/068ca7ae435a0be1c24d7f37ac8b8a2848cc6f38/src/theme.ts#L7) should be [gordonColors.primary.blueShades.50](https://github.com/gordon-cs/gordon-360-ui/blob/068ca7ae435a0be1c24d7f37ac8b8a2848cc6f38/src/theme.ts#L9). Though, obviously it's the center hue since it would save time and characters to not have to specify `primary.blueShades.500` so I get that.

I'm not exactly sure what your point here is: `gordonColors.primary.blue` and `gordonColors.primary.blueShades.500` are the same, with the former being the short-hand for the latter. Although in a more important sense the former is more accurate because it is actually the base color, and the other shades are simply variations on that theme color.

Reply:
1. fair enough
2. was more so referring to the event that `blueShade.50` became the new `primary.blue` which would satisfy the art definition of `shade` 

but neither here nor there, I made the adjustment to refer directly to the name in `theme.ts` which should setup a framework for future colors to be added 😎. Should show up on the next PR